core/executor/full_exit_nft_executor.go

```diff
@@ -230,5 +236,25 @@ func (e *FullExitNftExecutor) GenerateTxDetails() ([]*tx.TxDetail, error) {
 		CollectionNonce: exitAccount.CollectionNonce,
 	})
 
+	// create account empty delta
+	order++
+	accountOrder++
+	txDetails = append(txDetails, &tx.TxDetail{
+		AssetId:      types.EmptyAccountAssetId,
+		AssetType:    types.FungibleAssetType,
+		AccountIndex: txInfo.CreatorAccountIndex,
+		AccountName:  creatorAccount.AccountName,
+		Balance:      creatorAccount.AssetInfo[types.EmptyAccountAssetId].String(),
```

Comment:
what if the `creatorAccount.AssetInfo` is empty? you should refer deposit_nft_executor.go implementation